routes/videos.js

```diff
@@ -57,7 +61,7 @@ router.post('/', function (req, res) {
           detail: 'Internal Mongoose error while writing to database.'
         })
       }
-      res.json({ message: 'New session added!', data: video })
+      res.json({ message: 'New session added!', data: video, tokenId: token })
```

Comment:
We can still include the tokenId as part of data i think if we do a 
`video.tokenId = token within video.save`

I believe we can add a second param to the callback in video.save so its function (err, video)
In this case we have a new 'video' param in the scope of our callback that we are editing